docs/README.md

```diff
@@ -45,6 +46,20 @@ This repo provides some code samples for running a federated learning pipeline i
 | [examples](../examples) | Scripts and pipelines to run FL sample experiments |
 | [mlops](../mlops) | Provisioning scripts |
 
+
+### Tutorial on how to adapt the Factory code
```

Comment:
I would write this under a distinct markdown, with its own learning objectives and steps, I believe this is going to be a longer tutorial that will not fit in the README